### PR TITLE
feat(dop): Modify R&D Workflow

### PR DESCRIPTION
**Pull request by littlejiancc**
#### What this PR does / why we need it:
Modify R&D Workflow

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/gantt?id=325960&iterationID=1337&pId=0&type=REQUIREMENT)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |    Modify R&D Workflow          |
| 🇨🇳 中文    |     修改研发工作流       |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: .erda/migrations/dop/20220801-devflow.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    `org_id`                  BIGINT ( 20 ) NOT NULL DEFAULT 0 COMMENT 'orgID',
+    `org_name`                VARCHAR(50)  NOT NULL DEFAULT '' COMMENT 'orgName',
+    `app_id`                  BIGINT ( 20 ) NOT NULL DEFAULT 0 COMMENT 'projectID',
+    `app_name`                VARCHAR(50)  NOT NULL DEFAULT '' COMMENT 'projectName',
+    `creator`                 VARCHAR(36)  NOT NULL DEFAULT '' COMMENT 'creator',
+    `created_at`              datetime     NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'CREATED AT',
+    `updated_at`              datetime     NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'UPDATED AT',
+    `deleted_at`              BIGINT ( 20 ) NOT NULL DEFAULT '0' COMMENT 'deleted_at',
+    `branch`                  VARCHAR(50)  NOT NULL DEFAULT '' COMMENT 'branch',
+    `flow_rule_name`          VARCHAR(50)  NOT NULL DEFAULT '' COMMENT 'flowRuleName',
```
Comment (sfwn) [inline .erda/migrations/dop/20220801-devflow.sql:13]: what is `flow_rule_name`? why not use `rule_id` if have?

File: .erda/migrations/dop/20220801-devflow.sql [hunk 2]
```diff
@@ -0,0 +1,20 @@
+CREATE TABLE `erda_dev_flow`
+(
+    `id`                      VARCHAR(36)  NOT NULL DEFAULT '' COMMENT 'id',
+    `org_id`                  BIGINT ( 20 ) NOT NULL DEFAULT 0 COMMENT 'orgID',
```
Comment (sfwn) [inline .erda/migrations/dop/20220801-devflow.sql:4]: format your sql please. `(36)` vs `( 36 )`

**Review by sfwn — CHANGES_REQUESTED** [review]

File: .erda/migrations/dop/20220801-devflow.sql (hunk 1 above)
Comment (littlejiancc) [reply .erda/migrations/dop/20220801-devflow.sql:13]: flows are json without id

File: .erda/migrations/dop/20220801-devflow.sql (hunk 2 above)
Comment (littlejiancc) [reply .erda/migrations/dop/20220801-devflow.sql]: done

**Review by sfwn — APPROVED** [review]

**Pull request merged**